tests/pipelines/test_pipelines_text_generation.py

```diff
@@ -413,6 +431,7 @@ def run_pipeline_test(self, text_generator, _):
             "XGLMForCausalLM",
             "GPTNeoXForCausalLM",
             "FuyuForCausalLM",
+            "LlamaForCausalLM",
```

Comment:
Llama no longer raises exception for long generation, so I added it to exclude list in the test. This test was skipped on main.